fixing status option in documentation

bencart opened the pull request:
changing status option to statuses in the documentation

+label: docsite_pr

##### SUMMARY
Documentation currently has "status" option when it is actually "statuses"

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
aws_acm_facts

##### ADDITIONAL INFORMATION

bencart committed:
fixing status option in documentation

changing status option to statuses in the documentation

+label: docsite_pr

Akasurde commented on lib/ansible/modules/cloud/amazon/aws_acm_facts.py:
```diff
@@ -18,7 +18,7 @@
     aliases:
       - name
-  status:
+  statuses:
```
For backward compatibility and better UI experience, add 'statuses' as `aliases`.

acozine replied:
@Akasurde I don't see a `status` parameter in this module - only `statuses`.

Akasurde replied:
`status` option was available from the inception of this module, users might be using it. Changing directly to `statuses` will break existing playbooks. So I recommended using aliases instead of removing the parameter all together.

bencart replied:
@Akasurde you are incorrect. the option has always been statuses and the documentation has always been status.

Akasurde replied:
OK.

Akasurde commented:
@bencart Could you please address these issues - 
```
lib/ansible/modules/cloud/amazon/aws_acm_facts.py:0:0: E326 Argument 'statuses' in argument_spec defines choices as ([]) but documentation defines choices as ([['PENDING_VALIDATION'], ['ISSUED'], ['INACTIVE'], ['EXPIRED'], ['VALIDATION_TIMED_OUT']])
test/sanity/validate-modules/ignore.txt:3:1: A102 Remove since "lib/ansible/modules/cloud/amazon/aws_acm_facts.py" passes "E322" test
test/sanity/validate-modules/ignore.txt:4:1: A102 Remove since "lib/ansible/modules/cloud/amazon/aws_acm_facts.py" passes "E323" test
```

bencart committed:
changing statuses option choices to match AWS

Adding choices to the argument spec to match the documentation from AWS.
Adding 'REVOKED', 'FAILED' to statuses documentation to match implementation.

bencart committed:
Removing ignores for aws_acm_facts

Removing E322, E323 ignores for aws_acm_facts

acozine commented:
Merged despite a failing test - the failure is only due to a mismatch in the `version_added` field for the `statuses` parameter. The parameter has always been there; only the correction is being added in 2.8.

acozine commented:
Thanks @bencart for catching and fixing that error.
